### PR TITLE
doc: actions/act-runner: document running as a systemd service

### PR DESCRIPTION
**Pull request by nodiscc**
This documents running `act-runner` as a systemd service under a dedicated user account.

### CODE REVIEW

**Comment by nodiscc** [comment]
I would link to the specific issue on https://gitea.com/gitea/act_runner/issues/ requesting this, but `gitea.com` has been extremely slow for a few days, to the point of being unusable. Is this a known problem?

Edit: https://gitea.com/gitea/act_runner/issues/379

File: docs/content/usage/actions/act-runner.en-us.md [hunk 1]
```diff
@@ -262,6 +262,40 @@ The runner will fetch jobs from the Gitea instance and run them automatically.
...
+```ini
+[Unit]
+Description=Gitea Actions runner
+Documentation=https://gitea.com/gitea/act_runner
+After=docker.service
+
+[Service]
+ExecStart=/usr/local/bin/act-runner daemon --config /etc/act-runner/config.yaml
+ExecReload=/bin/kill -s HUP $MAINPID
+WorkingDirectory=/var/lib/act-runner
```
Comment (silverwind) [inline docs/content/usage/actions/act-runner.en-us.md]: Is a working directory a requirement? If not, I would recommend running in `/tmp` like [other daemons which don't require a working directory](https://github.com/redis/redis/blob/f7fa481156de051972dd49336fb04639bebdbe34/sentinel.conf#L68-L73).

Comment (techknowlogick) [reply docs/content/usage/actions/act-runner.en-us.md]: Working directory is required due to the .runner file.

**Comment by techknowlogick** [comment]
@nodiscc if you're comfortable saying, which continent are you located on, and what timezone/time of day are you accessing Gitea.com? I'm running operations for gitea.com and so your feedback is helpful/appreciated :)

**Comment by lng2020** [comment]
Is there a particular concern to use `-` instead of `_`?

**Comment by lunny** [comment]
It should be `act_runner`

**Comment by nodiscc** [comment]
> which continent are you located on, and what timezone/time of day are you accessing Gitea.com

Europe, GMT+1. I just issued a request to `https://gitea.com/gitea/act_runner/issues/nonexisting`, if that helps you look through the logs (page load took 13s, `https://gitea.com/assets/js/index.js?v=1.22.0~dev-233-g796ff26e0` didn't load at all, timeout)

> Is there a particular concern to use - instead of _

Most executables from Debian packages use `-` instead of `_`, all user/group names use `-`, also `_` is considered a word separator by many programs/editors, for consistency I always use `-` in file names unless there is a good reason.

```
$ ls /usr/bin/ | grep '\-' | wc -l
620
$ ls /usr/bin/ | grep '_' | grep -v x86|wc -l
105
$ grep '\-' /etc/passwd|wc -l
9
$ grep _ /etc/passwd | wc -l
0
```

Do you still want me to change it? Which parts (executable name? directory names? user name?)

**Comment by silverwind** [comment]
I personally find `-` more common and friendly than `_` and would even go as far as renaming `act_runner` to `act-runner`.

**Comment by techknowlogick** [comment]
Using `-` instead of `_` has implications on text selection and copy/paste. By using `_` in the binary name, it means when people double click on a part of the string for copying from docs etc, they will get the full `act_runner` instead of either just `act` or `runner`.

Regardless of which is better to use, as the built binary name uses `_` right now, I recommend that the docs should use that to stay consistent.

(thanks for the docs PRs btw, they are very appreciated 😄 )

**Comment by lunny** [comment]
Yes, we can have other PRs to discuss/do the renaming. But currently, please keep consistent on this PR.

**Comment by nodiscc** [comment]
Changed all occurrences of `act-runner` to `act_runner`.

**Review by lunny — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Pull request merged**